### ISSUE TITLE
Unable to edit field when keyboard nav cursor is on block

### ISSUE THREAD

**Issue by samelhusseini** [google/blockly#3744]
**Describe the bug**

Unable to edit a field by clicking on it when the keyboard nav cursor is on a block. We're able to when its on a field, why not block?
@alschmiedt FYI

**To Reproduce**

1. Go to the playground, drag a number of blocks together
2. Enable keyboard nav
3. Traverse around such that the cursor is on a block like so: 
<img width="481" alt="Screen Shot 2020-03-12 at 11 50 08 AM" src="https://user-images.githubusercontent.com/16690124/76556989-d06da380-6457-11ea-8c97-f58b498ff0d9.png">
4. Attempt to click on a field to edit it. Unable to click.

**Expected behavior**

Should be able to edit the field regardless what state we're in for keyboard nav.

**Screenshots**

Attached above.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Stack Traces**

```
Replace with error stack trace.
```

**Additional context**

**Comment by alschmiedt** [comment]
Fixed in #3745
